http/headers/Cross-Origin-Opener-Policy.json

```diff
@@ -40,6 +40,44 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "noopener-allow-popups": {
+          "__compat": {
+            "description": "noopener-allow-popups support",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy",
```

Comment:
Afaik we only put the `mdn_url` if there is a page specifically about this feature.

```suggestion
```